tests/Adapters/Microsoft.Bot.Builder.Adapters.Webex.TestBot/Startup.cs

```diff
@@ -15,7 +15,9 @@ public class Startup
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc(); //.SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_3_0);
```

Comment:
I don't think this line is needed in 3.1

Reply:
Removed from the Core 3.1 projects